tests/modeling/test_summarize_individuals.py

```diff
@@ -42,3 +44,10 @@ def test_tflite_not_installed(pheno_path, monkeypatch):
         monkeypatch.setitem(sys.modules, 'tflite_runtime', None)
         df = summarize_individuals([model])
         assert df['predicted_dofv'].isnull().all().all()
+
+
+def test_ofv(pheno_path):
+    parent_model = read_model(pheno_path)
+    candidate_model = Model()
```

Comment:
Can you also test with a model that has an empty `modelfit_results` object instead of `None`?

Reply:
So you want a model which is never fitted?

Reply:
Now we have a none modelfit_results:)